src/app/garfile/craft/index.njk

```diff
@@ -19,71 +19,71 @@
 {% block content %}
 
 <div class="govuk-grid-row">
-	<div class="govuk-grid-column-two-thirds">
 	{% if errors %}
 		{% include "common/templates/includes/errors.njk" %}
 	{% endif %}
-		<form id="page-form" action="/garfile/craft" autocomplete="off" class="form-group"
-      method="post" role="form">
+  <form id="page-form" action="/garfile/craft" autocomplete="off" class="form-group"
+    method="post" role="form">
+    <div class="govuk-grid-column-two-thirds">
 			<span class="govuk-caption-xl">{{__('section_count', 3, 8)}}</span>
 			<h1 class="govuk-heading-xl">{{__('heading_aircraft_details')}}</h1>
 
 			<h2 id="saved_aircraft" class="govuk-heading-l">{{__('title_aircraft')}}</h2>
 			<p class="govuk-body-l">{{__('caption_aircraft_details')}}</p>
-  </div>
-  <div class="govuk-grid-column-full">
-<table class="govuk-table">
-  <thead class="govuk-table__head">
-	<tr class="govuk-table__row">
-	  <th class="govuk-table__header" style="width: 5%;">&nbsp;</th>
-	  <th class="govuk-table__header">Aircraft registration</th>
-	  <th class="govuk-table__header">Aircraft type</th>
-	  <th class="govuk-table__header">Aircraft home port / location</th>
-	</tr>
-  </thead>
-  <tbody class="govuk-table__body">
-  {% if cookie.getSavedCraft().items | length %}
-    {% for craft in cookie.getSavedCraft().items %}
-      <tr class="govuk-table__row">
-        <div class="govuk-radios">
-        <td class="govuk-table__cell">
-          <div class="govuk-radios govuk-radios--inline">
-            <div class="govuk-radios__item">
-              <input class="govuk-radios__input" id="aircraft_{{ loop.index }}" name="addCraft" type="radio" value="{{craft.craftId}}">
-              <label class="govuk-label govuk-radios__label" for="aircraft_{{ loop.index }}"></label>
-            </div>
-          </div>
-        </td>
-        <td class="govuk-table__cell">{{ craft.registration }}</td>
-        <td class="govuk-table__cell">{{ craft.craftType }}</td>
-        <td class="govuk-table__cell">{{ craft.craftBase }}</td>
-      </tr>
-    {% endfor %}
-  {% else %}
-    <tr class="govuk-table__row">
-      <td colspan="4"><p class="govuk-inset-text">{{__('aircraft_no_records')}}</p></td>
-    </tr>
-  {% endif %}
-  </tbody>
-</table>
-{% if cookie.getSavedCraft().items | length %}
-  {{ pagination.build('page-form', pages, __)}}
-{% endif %}
-</div>
-<div class="govuk-grid-column-two-thirds">
-  {% if cookie.getSavedCraft().items | length %}
-    <div class="govuk-form-group">
-      <input id="saveCraft" class="govuk-button" type="submit" name="buttonClicked" value="Add to GAR"
-      onclick="sendAnalytics(event, 'Add craft to GAR - submit', 'click')">
     </div>
-  {% endif %}
+    <div class="govuk-grid-column-full">
```

Comment:
I think we may at odds regarding indentations, as this has happened a lot in this pull request